ffi/dotnet-idsdk/IoticsIdLib/src/AgentIdentity.cs

```diff
@@ -0,0 +1,9 @@
+namespace IOTICS;
+
+public class AgentIdentity(Identity factory, string seed, string keyName, string id, string did) : PartyIdentity(factory, seed, keyName, id, did)
```

Comment:
`PartyIdentity` is not a term used anywhere else in the codebase other than for C#. `IdentityBase`?
And e.g. Java there are no specific classes - just `Identity`. (I suppose there isn't time right now to consider having a similar look & feel for both :thinking: )